Huawei/hi6250/ANE/res/values/config.xml

```diff
@@ -227,6 +227,10 @@
 
     <!-- When true use the linux /dev/input/event subsystem to detect the switch changes
          on the headphone/microphone jack. When false use the older uevent framework. -->
+	
+    <!-- When set, it narrows the system user interface to fix problems with aspect ratio and
+         rounded corners. -->
+    <dimen name="rounded_corner_content_padding">58px</dimen>
```

Comment:
This is a SystemUI value, not a framework-res value, so it will not work.
But since Pie (Pie DR1?), you can set rounded_corner_radius in framework-res instead
So change to that. (If you tell me it really works this way, please tell, but I would have hard time believing you)

Reply:
_Tadi told me to copy paste this._
I mean, you can just make Samsung's SystemUI work on P20 lite.
Or we can create another System UI overlay for it.
Enabling overlay manually works fine.
My git and Android information ends here, sorry. :(